loveship/helpers/textLayout.tsx

```diff
@@ -1,4 +1,4 @@
-import css from '../assets/styles/scss/text-layout.scss';
```

Comment:
check could we delete this file, because it seams that isn't used?

Reply:
Deleted from loveship & promo. But it's still used in "uui".